File: May-18/17-05-18/python/02-Vowel-phobia.py
for _ in range(int(input())):
    data = input()
    
    cnt = 0
    for i in data:
        if(i == 'a' or
           i == 'e' or
           i == 'i' or
           i == 'o' or
           i == 'u' 
           ):
               cnt += 1
    print(cnt)
           

File: May-18/17-05-18/java/02-Vowel-phobia.java

import java.io.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(input.readLine());
        while(t-- > 0)
        {
            String str = input.readLine();
            int n = str.length();
            int cnt = 0;
            for(int i = 0; i < n; i++)
            {
                if(str.charAt(i) == 'a' ||
                   str.charAt(i) == 'e' ||
                   str.charAt(i) == 'i' ||
                   str.charAt(i) == 'o' ||
                   str.charAt(i) == 'u' 
                   )
                   cnt += 1;
            }
            System.out.println(cnt);
        }
    }
}
